src/libs/SelectionScraper/index.ts

```diff
@@ -64,17 +66,17 @@ const getHTMLOfSelection = () => {
             // and finally commonAncestorContainer.parentNode.closest('data-testid') is targeted dom.
             if (range.commonAncestorContainer instanceof HTMLElement) {
                 parent = range.commonAncestorContainer.closest(`[${tagAttribute}]`);
-            } else {
-                parent = range.commonAncestorContainer.parentNode.closest(`[${tagAttribute}]`);
+            } else if (range.commonAncestorContainer.parentNode) {
+                parent = (range.commonAncestorContainer.parentNode as HTMLElement).closest(`[${tagAttribute}]`);
             }
```

Comment:
```suggestion
            } else {
                parent = (range.commonAncestorContainer.parentNode as HTMLElement | null)?.closest(`[${tagAttribute}]`) ?? null;
            }
```

Would be safer if we do like this, in this way we are not changing conditional flow.